src/v2/Apps/NewForYou/NewForYouApp.tsx

```diff
@@ -1,24 +1,37 @@
 import React, { FC } from "react"
 import { MetaTags } from "v2/Components/MetaTags"
-import { Spacer, Text } from "@artsy/palette"
+import { Message, Spacer, Text } from "@artsy/palette"
 import { createFragmentContainer, graphql } from "react-relay"
 import { NewForYouApp_viewer } from "v2/__generated__/NewForYouApp_viewer.graphql"
-import { NewForYouArtworksGridFragmentContainer } from "v2/Apps/NewForYou/Components/NewForYouArtworksGrid"
+import { NewForYouArtworksGridRefetchContainer } from "v2/Apps/NewForYou/Components/NewForYouArtworksGrid"
+import { Link } from "v2/System/Router"
+import { useSystemContext } from "v2/System"
 
 interface NewForYouAppProps {
   viewer: NewForYouApp_viewer
 }
 
 export const NewForYouApp: FC<NewForYouAppProps> = ({ viewer }) => {
+  const { isLoggedIn } = useSystemContext()
+
   return (
     <>
       <Spacer mt={2} />
       <MetaTags title="New For You" />
       <Text variant="xl" mt={4}>
         New Works For You
       </Text>
-      <Spacer mt={4} />
-      {viewer && <NewForYouArtworksGridFragmentContainer viewer={viewer} />}
+      <Spacer mt={2} />
+      {!isLoggedIn && (
+        <Message variant="warning">
+          <Text>
+            Already have an account? <Link to="/login">Log in</Link> to see your
+            personalized recommendations.
+          </Text>
+        </Message>
+      )}
+      <Spacer mt={2} />
+      {viewer && <NewForYouArtworksGridRefetchContainer viewer={viewer} />}
```

Comment:
The initial spacer was mt={4}. Did half and half spacing and inserted the message in between. Personally, I think this looks better than 2x mt={4} since the Message area has such a significant amount of padding.

Reply:
That's fine but it counts as external margins if the viewer is null. We don't want this to have any external margin. I just mean to nest it inside of the conditional.

Reply:
I knew what you meant about the conditional. I think we're crossing wires. Will reach out via slack once everything else here is wrapped up.